src/util/strencodings.cpp

```diff
@@ -383,6 +383,25 @@ std::string Capitalize(std::string str)
     return str;
 }
 
+std::string ChallengeToStdString(const std::vector<uint8_t>& v)
```

Comment:
move ChallengeToStdString to src/util/strencodings.cpp for later availability in main window.